components/dialog/dialog-fullscreen.js

```diff
@@ -95,12 +161,26 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 		this.async = false;
 		this._autoSize = false;
 		this._hasFooterContent = false;
+		this._iconType = 'tier1:close-large-thick';
+		this._handleResize = this._handleResize.bind(this);
+		this._mediaQueryList = window.matchMedia('(max-width: 616px)');
```

Comment:
While we aren't expecting a lot of dialogs on the page, the 
```javascript
this._mediaQueryList = window.matchMedia('(max-width: 616px)');
```
could be moved outside of the class, since we don't really need a different instance for every dialog.